style.css

```diff
@@ -0,0 +1,90 @@
+@media (min-width: 200px) and (max-width: 699px) {
+  body {
+    background-image: url("images/header-shapes mobile.png");
+    background-repeat: no-repeat;
+    background-size: 100%;
+  }
+}
+
+@media (min-width: 700px) {
+  body {
+    background-image: url("images/Header bg@2x.png");
+    background-repeat: no-repeat;
+  }
+
+  .headline {
+    align-items: center;
+  }
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+}
+
+.headline-heading {
+  width: 327px;
+  height: 150px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  margin-top: 128px;
+  margin-left: 24px;
+  color: #172b4d;
+  align-items: center;
+}

```

Comment:
- [ ] I would suggest you please use the Font Poppins as you use it but you didn't import it from google font. please import font and use it